### DISCUSSION TITLE
Why is Mattermost template ignored?

### DISCUSSION THREAD

**Discussion by hnykda** [coollabsio/coolify#3254] [Q&A] [answered]
It seems I [reimplemented the wheel](https://github.com/coollabsio/coolify/discussions/3087) - mattermost template is here: https://github.com/coollabsio/coolify/blob/main/templates/compose/mattermost.yaml, but it has `ignored: true`. Why is that?

**Comment by peaklabs-dev** [comment, marked as answer]
Because I did not work before. I fixed it in the last version and you can now use it.
